pkg/mock/mockoption/mockoption.go

```diff
@@ -32,7 +32,7 @@ const (
 	defaultReplicaScheduleLimit        = 64
 	defaultMergeScheduleLimit          = 8
 	defaultHotRegionScheduleLimit      = 4
-	defaultStoreBalanceRate            = 60
+	defaultStoreLimit                  = 64
```

Comment:
Why change this to `64`?

Reply:
I don't know, maybe this is a typo. Change it back?

Reply:
Seems `60` is a more meaningful value

Reply:
OK, I'll change it back.